website/docs/getting-started/overview.mdx

```diff
@@ -9,8 +9,8 @@ image: https://miro.medium.com/max/1400/1*iZfQ7ST9rd7McrHvvVA-BA@2x.png
 Working with Mesh means dealing with 4 main concepts: _Sources_, _Handlers_, _Transforms_ and _Unified Schema_:
 
 - In Mesh, a sub-service (GraphQL API, REST API) is called a _Source_.
-- Sources are translated to GraphQL Schema with the appropriate _Handler_.
-- Then, all Sources' GraphQL Schema are merged into a final _Unified Schema_.
+- Sources are translated to GraphQL Schemas with the appropriate _Handler_.
+- All Sources' GraphQL Schema are merged into a final _Unified Schema_.
 - Finally, if applicable, configured transformations, called _Transforms_, are applied to the _Unified Schema_.
```

Comment:
Transforms can be defined in root or source level.

Reply:
I know but this difference will be covered in the "Advanced" section, on the page "Order of transforms, Sources' Schema, and unified Schema"

I feel that talking about both the  Sources' schemas and Unified Schema is confusing for the Getting Started section.